src/index.ts

```diff
@@ -1,13 +1,80 @@
+
+// Creates a div container
 const container = document.createElement('div');
 container.style.position = 'fixed';
+container.style.bottom = '64px'; // padding from bottom
+container.style.right = '64px'; // padding from the right
+container.style.zIndex = '1000'; // Ensure the icon is above other elements
+
+// Creates a div element for the icon
+const icon = document.createElement('div');
+
+// Sets the inner HTML of the icon to the SVG content
+icon.innerHTML = svgContent;
+
+// Sets CSS properties for the icon
+icon.style.width = '40px'; //  This sets the width of the icon
+icon.style.height = 'auto'; //  This maintains aspect ratio
+icon.style.cursor = 'pointer'; // This changes the cursor while hoovering over the icon
 
-// Place the widget in the bottom right corner
-container.style.bottom = '30px';
-container.style.right = '30px';
+// event listener to do something when clicked
+icon.addEventListener('click', () => {
+  alert('Icon clicked!');
+});
 
-// TODO: Rather than just displaying the config, draw an icon in the bottom right corner
-container.innerHTML = JSON.stringify(window.SmocConfig);
+// Sets a class for the icon for additional styling
+icon.className = 'custom-icon';
+
+// Append the icon to the container
+container.appendChild(icon);
 
 // Add the container as the last child of the <body> element.
 document.body.appendChild(container);
+
+// Function to change SVG color
+function setSVGColor(svgElement: SVGElement, color: string): void {
```

Comment:
This works, but there is a simpler/cleaner way. Rather than modifying the SVG elements' `style` property, style it with a `class`.

Changing the colour is then done by modifying the style instead. See https://stackoverflow.com/questions/524696/how-to-create-a-style-tag-with-javascript for an example.

It's up to you whether you want to change this.